pkgs/applications/misc/flashprint/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, lib, libGLU, qtbase, fetchurl, dpkg, autoPatchelfHook, wrapQtAppsHook }:
+
+let
+  version = "5.7.1";
+
+in stdenv.mkDerivation {
+  pname = "flashprint";
+  inherit version;
+
+  src = fetchurl {
+    url = "http://www.ishare3d.com/3dapp/public/FlashPrint-5/FlashPrint/flashprint5_${version}_amd64.deb";
```

Comment:
Is it fine that the source is not from Flashforge directly?

Reply:
This does have the advantage of not having some obscure hash url like in #241474.

Reply:
The AUR package in Arch Linux uses this ishare3d.com URL since 2017, as can be seen here: https://aur.archlinux.org/cgit/aur.git/commit/?h=flashprint&id=b9ab733523c6ec2e38f552ed6a8f7959ddf32942

I'm using this AUR package for 2 years now, and I think it is OK. Wonder what others think.